urls.py

```diff
@@ -13,7 +13,8 @@
     #url(r'^about/(?P<pagename>[a-z\-]+)/$', 'kaeru.views.about_view'),
     url(r'^documentation/$' , 'kaeru.views.documentation_view'), 
     url(r'^login/$' , 'kaeru.views.login_view'), 
-    url(r'^logout/$' , 'kaeru.views.logout_view'), 
-    url(r'^people/$' , 'kaeru.views.people_view'),
-    url(r'^tutorial/$' , 'kaeru.views.tutorial_view'), 
+    url(r'^logout/$' , 'kaeru.views.logout_view'),
+    url(r'^signup/$' , 'kaeru.views.signup_view'),
+    # url(r'^people/$' , 'kaeru.views.people_view'),
```

Comment:
Oh, it's when yesterday's debug, in order to go to admin site we commented 2 lines. In fact it should be tutorial's page that prevent us from going to admin site -- and maybe we should only comment this 1 line out?
So what I should only do is to revert the change to people page?